Bootstrap.mak

```diff
@@ -88,7 +88,7 @@ mingw: mingw-clean
 	$(CC) -o build/bootstrap/premake_bootstrap -DPREMAKE_NO_BUILTIN_SCRIPTS -I"$(LUA_DIR)" -I"$(LUASHIM_DIR)" $(SRC) -lole32 -lversion
 	./build/bootstrap/premake_bootstrap embed
 	./build/bootstrap/premake_bootstrap --arch=$(PLATFORM) --os=windows --to=build/bootstrap --cc=mingw gmake2
-	$(MAKE) -C build/bootstrap config=$(CONFIG)_$(PLATFORM)
+	$(MAKE) -C build/bootstrap config=$(CONFIG)_$(PLATFORM:x86=win32)
```

Comment:
Not familiar with this mingw. Can you explain what this change does?

Reply:
gcc --version
gcc (x86_64-posix-seh-rev0, Built by MinGW-W64 project) 8.1.0

Once the core source code has been cloned, I started the first Premake executable:

mingw32-make -f Bootstrap.mak mingw CC=gcc

error stop

Makefile:34: *** "invalid configuration release_x86". Stop.
mingw32-make: *** [Bootstrap.mak:91: mingw] Error 2

config=release_x86 not found  into build\bootstrap\Makefile autogenerated by Premake
but configuration config=release_win32

this fixes error
$(MAKE) -C build/bootstrap config=$(CONFIG)_$(PLATFORM:x86=win32)

while it's ok for PLATFORM=x64
mingw32-make -f Bootstrap.mak mingw PLATFORM=x64 CC=gcc

Reply:
So does :x86=win32 just substitute "x86" for "win32"?

Reply:
Yes, like target windows into Bootstrap.mak

windows: windows-base
	devenv .\build\bootstrap\Premake5.sln /Upgrade
	devenv .\build\bootstrap\Premake5.sln /Build "$(CONFIG)|$(PLATFORM:x86=win32)"

Reply:
Appreciate you helping me understand the change. +1 and merging.

Reply:
Docs for further explanation https://www.gnu.org/software/make/manual/html_node/Substitution-Refs.html#Substitution-Refs